code/modules/customitems/item_defines.dm

```diff
@@ -654,6 +655,24 @@ All custom items with worn sprites must follow the contained sprite system: http
 	else
 		..()
 
+/obj/item/fluff/tokash_spear/MouseDrop(mob/user as mob)
+	if((user == usr && (!( usr.restrained() ) && (!( usr.stat ) && (usr.contents.Find(src) || in_range(src, usr))))))
+		if(!istype(usr, /mob/living/carbon/slime) && !istype(usr, /mob/living/simple_animal))
+			if( !usr.get_active_hand() )		//if active hand is empty
+				var/mob/living/carbon/human/H = user
+				var/obj/item/organ/external/temp = H.organs_by_name[BP_R_HAND]
+
+				if (H.hand)
+					temp = H.organs_by_name[BP_L_HAND]
+				if(temp && !temp.is_usable())
```

Comment:
i think there's a proc for this. (`use_check_and_message()`)

Reply:
I'm not entire sure what you mean by this. Are you saying that 660 through 666 are unnecessary or that a different line using use_check_and_message() should be used?